### PR TITLE
chore: Update description for fastlane metadata

### PR DESCRIPTION
**Pull request by liveHarshit**
Part of #2314 
According to https://docs.fastlane.tools/getting-started/android/setup/

### CODE REVIEW

**Comment by liveHarshit** [comment]
> That's for apple

Updated

File: fastlane/metadata/android/en-US/full_description.txt [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
-- Users have also the privilege to favourite events in the app
\ No newline at end of file
+- Functionality to filter out events by date, time, location and event name
+- Users can buy tickets and register as attendees for any event
+- Pay for their orders via PayPal and Stripe
+- All important event details such as location, date, and timing of the event can be viewed
+- Users can view all the tickets bought for an event with their status
+- Easy check-in using QR code for Tickets and see check-in timings
+- Users can view similar events
+- Users have also the privilege to a favorite event
```
Comment (ShridharGoel) [inline fastlane/metadata/android/en-US/full_description.txt]: Check this line. Should either be 'Users have the privilege to favourite an event' or 'Users have the privilege to mark an event as favourite'.

File: fastlane/metadata/android/en-US/full_description.txt [hunk 2]
```diff
@@ -1,13 +1,14 @@
-With EventYay Attendee, you can browse through different events happening nearby or all over the world. You can get detailed information about the events and can even buy event tickets.
-The app also offers other cool features such as similar events and easy check-in using QR code. If that’s not enough, the app also allows you to favorite different events and all of them can be accessed from favorite section.
-
-Eventyay Attendee App provides features for users like:-
+With EventYay Attendee, you can browse through different events happening nearby or anywhere in the world. You can get detailed information about the events and buy tickets. The app also offers other features such as displaying ticket details and easy check-in using QR code. If that’s not enough, the app also allows you to favorite events and search events by their name.
```
Comment (ShridharGoel) [inline fastlane/metadata/android/en-US/full_description.txt]: `Eventyay`. `Y` shouldn't be capital.

Comment (liveHarshit) [reply fastlane/metadata/android/en-US/full_description.txt]: I think it should be, Event+yay, here event description is started and the exact name is with the app.

File: fastlane/metadata/android/en-US/full_description.txt (hunk 1 above)
Comment (liveHarshit) [reply fastlane/metadata/android/en-US/full_description.txt]: > Check this line. Should either be 'Users have the privilege to favourite an event' or 'Users have the privilege to mark an event as favourite'.

favorite*

Comment (liveHarshit) [reply fastlane/metadata/android/en-US/full_description.txt]: @ShridharGoel Updated :+1:

File: fastlane/metadata/android/en-US/full_description.txt (hunk 2 above)
Comment (ShridharGoel) [reply fastlane/metadata/android/en-US/full_description.txt]: But the app name is Eventyay and not EventYay

Comment (liveHarshit) [reply fastlane/metadata/android/en-US/full_description.txt]: Can you please check out the tag here: https://blog.fossasia.org/tag/eventyay/

Comment (ShridharGoel) [reply fastlane/metadata/android/en-US/full_description.txt]: Ok

**Pull request merged**